eng/restore-toolset.ps1

```diff
@@ -1,32 +1,12 @@
 function InitializeCustomSDKToolset {    
-  if ($fullMSBuild) {
+  if ($msbuildEngine -eq 'vs') {
     if (!($env:VSInstallDir)) {
-      $env:VSInstallDir = LocateVisualStudio
+      $env:VSInstallDir = Join-Path $env:VSSDKInstall ".."
     }
 
     $env:DOTNET_SDK_TEST_MSBUILD_PATH = Join-Path $env:VSInstallDir "MSBuild\15.0\Bin\msbuild.exe"
```

Comment:
Oh, nevermind previous comment. This is what needs to happen and looks like it still is.